Sources/Commands/SwiftPackageTool.swift

```diff
@@ -30,44 +30,44 @@ import func POSIX.chdir
 extension PackageToolOptions: XcodeprojOptions {}
 
 private enum Mode: Argument, Equatable, CustomStringConvertible {
-    case Init(InitMode)
+    case initPackage
     case doctor
-    case showDependencies(ShowDependenciesMode)
+    case showDependencies
     case fetch
     case update
     case usage
     case version
-    case generateXcodeproj(String?)
-    case dumpPackage(String?)
+    case generateXcodeproj
+    case dumpPackage
 
     init?(argument: String, pop: () -> String?) throws {
         switch argument {
         case "init", "initialize":
-            self = try .Init(InitMode(pop()))
+            self = .initPackage
         case "doctor":
             self = .doctor
-        case "show-dependencies", "-D":
-            self = try .showDependencies(ShowDependenciesMode(pop()))
+        case "show-dependencies", "show-deps":
```

Comment:
Actually, now I disagree — show-dependencies is a command, not an option, and this would be the first command that we would abbreviate with a dash.  I don't like the inconsistency of that at all.